okhttp/src/main/java/okhttp3/internal/http2/Header.kt

```diff
@@ -26,16 +25,14 @@ data class Header(
   /** Value in UTF-8 encoding. */
   @JvmField val value: ByteString
```

Comment:
Can you default value to "" to cleanup definitions below?

Reply:
I don’t think it’s a good default for `Header` because only Hpack wants empty strings here.